### ISSUE TITLE
Crash when opening Git mode on Linux

### ISSUE THREAD

**Issue by nyanpasu64** [francescmm/GitQlientPlugin#4]
Qt Creator 4.11.0, from Manjaro Testing branch.

I downloaded libGitQlientPlugin_1.0.0-qtc4.11.x.so.

When I open Qt Creator, open a project, and switch to Git panel, I get the following terminal error, then Qt Creator exits:

```
"/home/nyanpasu64/Dropbox/encrypted/code/exotracker"
qtcreator: symbol lookup error: /home/nyanpasu64/.local/share/data/QtProject/qtcreator/plugins/4.11.0/libGitQlientPlugin_1.0.0-qtc4.11.x.so: undefined symbol: _ZdlPvm, version Qt_5
ConnectionServer error: "QLocalSocket: Remote closed" "/tmp/QtCreator-CLqyde/ClangBackEnd-18652"
```

The first time, Qt Creator froze (possibly due to mechanical disk iowait). I launched Qt Creator from .desktop file and could not see what was printed to the terminal.

**Comment by francescmaestre** [comment]
I think it can be a problem with the new Qt Creator 4.11 and the build system. I'll try to build a new version ASAP! :)

**Comment by francescmaestre** [comment]
Hi, I've uploaded a new version of libGitQlientPlugin_1.0.0-qtc4.11.x.so for linux and QtCreator 4.11. I tested locally and so far it works.

Could you double check it? :)

**Comment by sandervalcke** [comment]
I'm getting the same error for the creator 4.10.2 version (also manjaro). Could you do the same for that plugin?

**Comment by francescmaestre** [comment]
I would have to check what could be happening since I have 4.10.2 working in three different places... could you paste some system info?

Gcc version, Qt version, QtCreator version. Where is that Qt Creator build from? Arch linux package? Qt Project download?

Can you please run Qt Creator from GDB and generate a full backtrace?

Start from a terminal with
`gdb <path_to_qtcreator_binary>`
In GDB then run with
`run`
After it crashed generate backtrace with
`thread apply all bt`
and attach the result here.

If I find time today/tomorrow I'll try Manjaro to see what's going on, but I would need more info about how QtCreator was built :)

**Comment by nyanpasu64** [comment]
> After it crashed generate backtrace with
thread apply all bt
and attach the result here.

There was no crash to obtain a backtrace from. The program appears to have exited semi-cleanly.

EDIT:

I obtained Qt Creator through `sudo pacman -S qtcreator`. I used to have a self-build 4.11 RC1, but switched to 4.11 release.

Qt Creator 4.11.0
Based on Qt 5.13.2 (GCC 9.2.0, 64 bit)

Operating System: Manjaro Linux 
KDE Plasma Version: 5.17.4
KDE Frameworks Version: 5.64.0
Qt Version: 5.13.2
Kernel Version: 5.4.3-1-MANJARO
OS Type: 64-bit
